packages/create-bison-app/template/graphql/schema.ts

```diff
@@ -13,6 +13,7 @@ export const schema = makeSchema({
   types,
   plugins: [
     fieldAuthorizePlugin(),
+    declarativeWrappingPlugin(),
```

Comment:
This was required to add to the configuration in order to satisfy some of the current schema definitions we had.
https://nexusjs.org/docs/plugins/declarativeWrapping